test/e2e/framework/test_context.go

```diff
@@ -208,7 +208,7 @@ type NodeTestContextType struct {
 	// PrepullImages indicates whether node e2e framework should prepull images.
 	PrepullImages bool
 	// KubeletConfig is the kubelet configuration the test is running against.
-	KubeletConfig kubeletconfig.KubeletConfiguration
+	KubeletConfig kubeletconfigv1beta1.KubeletConfiguration
```

Comment:
This change is making an error on e2e_node test side as 
```
# k8s.io/kubernetes/test/e2e_node
vet: test/e2e_node/runtimeclass_test.go:97:9: invalid operation: operator ! not defined for framework.TestContext.KubeletConfig.CgroupsPerQOS (variable of type *bool)
make[1]: *** [Makefile:386: vet] Error 1
```
because of the difference of CgroupsPerQOS between 2 repos.
- k/kubelet: *bool as https://github.com/kubernetes/kubelet/blob/368f45b4d61707ccd2db0e9d12a1fd0c1ee24f64/config/v1beta1/types.go#L403
- k/kubernetes: bool as https://github.com/kubernetes/kubernetes/blob/4bc907f3c956ffb808fdcfb4f8fc07825b8d2339/pkg/kubelet/apis/config/types.go#L203

I think we need to update caller sides also in this PR to match this difference.

Reply:
right.
i will update it
/hold 
for ci

Reply:
@tanjunchen could we leave this part for another PR?
The patch for `test/e2e/framework/pods.go` is pretty good.
I would recommend to revert the changes in this file for this PR.
We can pick this back up in another one.
Plus it helps keep things organized.

Reply:
thanks , ok i will revert  this file

Reply:
or copy and paste it :sweat_smile: 
but the intent in this file is good enough for its own PR